### ISSUE TITLE
Add species images

### ISSUE THREAD

**Issue by nathankota** [agrc/roadkill-mobile#63]
> Can we show images next to each species selection for reports and pick ups?

**Comment by stdavis** [comment]
Waiting on images from DWR. I guess that we're not sure on the size yet.

**Comment by nathankota** [comment]
Can we insert an image placeholder once the list is verticalized (#142), then let them know the size to create ([#141](https://github.com/agrc/roadkill-mobile/issues/141))?

**Comment by stdavis** [comment]
![image](https://user-images.githubusercontent.com/1326248/144148770-d50e0588-4e3b-4d75-a525-8f246549c12a.png)

This will take more effort once we start getting images from them. I'll need to:
- [ ] create a bucket to put them in
- [ ] create an API endpoint for retrieving them
- [ ] come up with a system for Jessie to maintain them

But before I go that deep, I'd like the client to sign off on how they look in the app. I'm a little worried that they are going to ask to be able to click on them and see a larger version or something else.

**Comment by nathankota** [comment]
They are just starting to put images [here](https://drive.google.com/drive/folders/1U5nvsX6IM4O5pYEkxFMjXLDJnoIffJ32) at the moment, and I have just made them aware of the size (150 x 150 px) of the v515 layout.

**Comment by stdavis** [comment]
Here's a link to the upload instructions: https://github.com/agrc/roadkill-mobile/tree/staging#identification-images. I've sent these to Jessie and asked her for feedback and to verify that she has access to the bucket.

**Comment by stdavis** [comment]
![image](https://user-images.githubusercontent.com/1326248/145604769-aac8da23-7699-4c44-90a4-83f9d7634209.png)

**Comment by stdavis** [comment]
BTW - Have you noticed that they have already populated a bunch of them? You can see them in the report form by searching for common name.

**Comment by nathankota** [comment]
Yes, they're looking good!

**Comment by stdavis** [comment]
Yes, they populate automatically. I have a six-hour cache built into them so it may take a bit for them to show up.
